### PR TITLE
Additional updates, Jun 9, 2024.

### PR DESCRIPTION
**Pull request by cryptoquick**

### CODE REVIEW

File: bip-p2qrh.mediawiki [hunk 1]
```diff
@@ -14,18 +14,22 @@
...
 
 The vulnerability of existing bitcoin addresses is investigated in [https://www2.deloitte.com/nl/nl/pages/innovatie/artikelen/quantum-computers-and-the-bitcoin-blockchain.html this Deloitte report]. The report estimates that in 2020 approximately 25% of the bitcoin supply is held within addresses vulnerable to quantum attack.
 
+Currently, any address is vulnerable to quantum attack the moment it's spent from and when that transaction is submitted publicly to the mempool. P2QRH protects transactions submitted to the mempool, and preserves the free market by reducing the need for private, out of band transactions directly to mining pools.
```
Comment (CrewsControlSolutions) [inline bip-p2qrh.mediawiki:31]: Tweaked second sentence:

> P2QRH protects transactions submitted to the mempool and helps preserve the free market by reducing the need for private, out-of-band transactions directly to mining pools.

Comment (cryptoquick) [reply bip-p2qrh.mediawiki]: Updated, thank you

File: bip-p2qrh.mediawiki [hunk 2]
```diff
@@ -48,6 +52,8 @@ Should quantum advantage manifest, a convention is proposed in spending the full
 
 As an interesting aside, coinbase outputs to P2PK keys go as far as block 200,000, so it's possible there are between 1-2 million coins that are vulnerable from the first epoch. These coins can be considered "Satoshi's Shield." Any addresses with a balance of less than the original block subsidy of 50 coins can be considered incentive incompatible to capture until all of these are mined.
 
+In comparison, [https://ethresear.ch/t/how-to-hard-fork-to-save-most-users-funds-in-a-quantum-emergency/18901 Vitalik Buterin's proposed solution] for quantum emergency in Ethereum is to hard fork the chain, revert all blocks after a sufficient amount of theft, and use STARKs based on BIP-32 seeds to act as the authoritative secret when signing. This approach is deemed far too heavy-handed for Bitcoin.
```
Comment (CrewsControlSolutions) [inline bip-p2qrh.mediawiki:55]: I would tweak the paragraph to something similar to this:

_Lastly, it is worth noting by way of comparison that [https://ethresear.ch/t/how-to-hard-fork-to-save-most-users-funds-in-a-quantum-emergency/18901 Vitalik Buterin's proposed solution] in an Ethereum quantum emergency is quite different from the approach in this BIP.  His plan involves a hard fork of the chain, reverting all blocks after a sufficient amount of theft, and using STARKs based on BIP-32 seeds to act as the authoritative secret when signing.  These measures are deemed far too heavy-handed for Bitcoin._

File: bip-p2qrh.mediawiki [hunk 3]
```diff
@@ -140,6 +149,4 @@ To help implementors understand updates to this BIP, we keep a list of substanti
 
 == Acknowledgements ==
 
-TBD
-
-Much gratitude to my co-founder, Kyle Crews for proofreading and editing, and also to Dave C. who provided the name, QuBit.
+Much gratitude to my co-founder, Kyle Crews for proofreading and editing, to David C. who provided the name, QuBit. Same for those who took the time to review and contribute, including AB, (and hopefully many more).
```
Comment (CrewsControlSolutions) [inline bip-p2qrh.mediawiki:152]: nit pic on wording:

Much gratitude to my co-founder, Kyle Crews for proofreading and editing, and to David C. who suggested the name "QuBit." Thank you as well to those who took the time to review and contribute, including AB (and hopefully many more).

**Pull request merged**